python/pyspark/ml/feature.py

```diff
@@ -1110,6 +1110,80 @@ class PCAModel(JavaModel):
     """
 
 
+@inherit_doc
+class RFormula(JavaEstimator, HasFeaturesCol, HasLabelCol):
+    """
+    .. note:: Experimental
+
+    Implements the transforms required for fitting a dataset against an
+    R model formula. Currently we support a limited subset of the R
+    operators, including '~', '+', '-', and '.'. Also see the R formula
```

Comment:
Would it be nice to have an example about the dot operator? I could not find one in the RFormulaSuite as well.